packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -28,6 +30,18 @@ export const barChartSchema = {
     stacked: {
       type: 'boolean',
     },
+    showAnnotations: {
+      type: 'boolean',
+    },
+    chartDatasetColors: {
```

Comment:
Could you please refactor this definition out into its own constant?  It's shared between bar charts and timeseries charts.